ui/src/assets/chat_page.scss

```diff
@@ -0,0 +1,110 @@
+/* Make the main container a flex column that fills the available height */
+.chat-container {
```

Comment:
All classes should be prefixed with `pf-` and please follow BEM (https://getbem.com/) class naming conventions.